examples/guestbook/README.md

```diff
@@ -83,15 +158,17 @@ kind: ReplicationController
 metadata:
   name: redis-master
   labels:
-    name: redis-master
+    app: redis
```

Comment:
These labels should be applied automatically from the labels in the pod template. I suggest leaving them here, but adding a comment to that effect.